torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -112,6 +112,65 @@ def replace_no_op(node, replace_input_index):
             replace_no_op(node, 0)
 
 
+@torch.utils._python_dispatch._disable_current_modes()
+def remove_redundant_complex_views(gm: torch.fx.GraphModule):
+    """
+    Removes redundant complex views by reusing existing ones.
+    """
+
+    # A dictionary mapping a tensor to all aliased views.
+    aliases = {}
```

Comment:
aliases seems like the wrong name for this, since it is only tracking complex view ops -- not all aliases

Reply:
changed to `complex_views`.